src/main/java/io/github/debuggyteam/tablesaw/client/TableSawClient.java

```diff
@@ -25,9 +28,15 @@
 import static io.github.debuggyteam.tablesaw.TableSaw.TABLESAW;
 
 public class TableSawClient implements ClientModInitializer {
-
+	
+	public static ClientSettings config;
+	
 	@Override
 	public void onInitializeClient(ModContainer mod) {
+		
+		config = QuiltConfig.create(TableSaw.MODID, "client", ClientSettings.class);
+		
+		
 		HandledScreens.register(TableSaw.TABLESAW_SCREEN_HANDLER, (TableSawScreenHandler gui, PlayerInventory inventory, Text title) -> new TableSawScreen(gui, inventory, title));
```

Comment:
There's extra newlines in TableSawClient.java and TableSawScreen (unless those only show up in the "review" page)